README.md

```diff
@@ -2,39 +2,52 @@
 
 [![PyPI Release](https://github.com/scailable/sclblpy/workflows/PyPI%20Release/badge.svg)](https://pypi.org/project/sclblpy/)
 
-`sclblpy` is the core python package provided by Scailable to convert models fit in python to WebAssembly and
-open them up as a REST endpoint.
 
-Currently the package supports the upload of fitted `sklearn` model objects, and it supports uploading `.onnx` models
-(by specifying a path to the `.onnx` file). The package can also be used to manage assignments of models to 
-registered devices. 
+Changelog: the package is currently undergoing a major rework. We try to keep the below up to date, but some features might be stubs. 
+Most relevant is the removal of support for Scikit learn in favor of focussing fully on ONNX.
```

Comment:
Maybe: "As Scailable has changed its focus from Scikit Learn to ONNX, Scikit Learn support has been removed. "

Reply:
added